DiamondLightSource/SynchWeb
# LIMS-1570: Remove unused app file

**JIRA ticket**: [LIMS-1570](https://jira.diamond.ac.uk/browse/LIMS-1570)

**Summary**:

Since the app file is not being used, and it features some unmaintained code, I figured it would be best to remove it.

Alongside it, a few unused libraries are also being removed.

**Changes**:
- Remove unused dependencies
- Remove dead app.js file and related files (superseded by Vue views)

**To test**:
- Go to any page, ensure the sidebar still displays all expected links
- Log in, ensure the log in page still works
- Ensure the navbar still has all expected icons
![image](https://github.com/user-attachments/assets/e45d00b4-6bb4-4f5b-8042-c23889edc415)
